tests/db/migration_tests/test_v202206290_to_v202208180.py

```diff
@@ -34,13 +34,13 @@ async def test_column_add(
     """
 
     # Assert state before running the DB migration script
-    assert "partial_base" not in (await get_columns_in_db_table(ConfigurationModel.table_name()))
+    assert "partial_base" not in (await get_columns_in_db_table(ConfigurationModel.table_name(include_schema=False)))
```

Comment:
Why not make the `ConfigurationModel.table_name`  backward compatible by defaulting to false?

Reply:
I was going to make the same remark.